.eslintrc.json

```diff
@@ -6,9 +6,9 @@
     "jasmine": true,
     "es6": true
   },
-  "plugins": ["promise"],
+  "plugins": ["promise", "@typescript-eslint/eslint-plugin"],
```

Comment:
nice, so this is now a go

Reply:
Yeah, this is the official, ran my MS, one too